scripts/find_unique_isoform.py

```diff
@@ -7,17 +7,24 @@
 import pysam
 import edlib
 import numpy
+import logging
 from Bio import SeqIO
 
+logging.basicConfig(
```

Comment:
This is a comment, not a suggestion for a change. You can set up the config in the script, but I typically like to leave this to a higher level in the package. If you just do this:

```
logging.getLogger(__name__).addHandler(logging.NullHandler())
```
Then you can log, and either a user, or at some other level in the package, the config can be set up

Reply:
In fact, you probably do want to do this, because right now the log level is hard coded